app/src/forms/common/models/tables/user.js

```diff
@@ -40,11 +40,13 @@ class User extends Timestamps(Model) {
           query.where('idpCode', value);
         }
       },
-      filterUsername(query, value, exact = false) {
+      filterUsername(query, value, exact = false, caseSensitive = true) {
         if (value) {
-          if (exact) query.where('username', value);
+          if (exact && caseSensitive) query.where('username', value);
           // ilike is postgres case insensitive like
-          else query.where('username', 'ilike', `%${value}%`);
+          else if (exact && !caseSensitive) query.where('username', 'ilike', String(value).toLowerCase());
+          else if (!exact && caseSensitive) query.where('username', 'ilike', `%${value}%`);
+          else query.where('username', 'ilike', `%${String(value).toLowerCase()}%`);
```

Comment:
If the comment on line 46 is correct, then the `.toLowerCase()` isn't needed, is it? And if it isn't needed, then don't lines 48 and 49 become the same thing? If so, then is one of them incorrect? (apologies for the quick and vague comment - haven't had time to play around with this).

Reply:
I think the line 46 talks about `lower` keyword where we put in column name for the database query, value still needs to be applied with `.toLowerCase()`.